philippa-choong/week5/mon/game/app/controllers/rps_controller.rb

```diff
@@ -0,0 +1,35 @@
+class RpsController < ApplicationController
+
+    def throw_selection
+
+    end
+
+    def throw_verdict
+
+        @user_selection = params[:throw].downcase
+        @rand_number = rand(0 .. 2)
+        array = ["rock", "paper","scissors"]
+        @comp_select = array[@rand_number]
+
+
+        conditions = {
+        "Rock" => "Scissors", 
+        "Paper" =>  "Rock",
+        "Scissors" => "Paper"
+        }
+
```

Comment:
You win condition does not work :'(

The downcase you have put on line 9 conflicts with your conditions and how they were written out. 

If i enter `Paper`, it will be turned into lowercase from line 9 and goes through `condition["paper"]` and there is no such thing, so it will always run line 27 to 29.